web/client/codechecker_client/credential_manager.py

```diff
@@ -64,39 +64,50 @@ def __init__(self):
         user_home = os.path.expanduser("~")
         session_cfg_file = os.path.join(user_home,
                                         ".codechecker.passwords.json")
-        LOG.debug(session_cfg_file)
+        LOG.info("Checking local passwords or tokens in %s", session_cfg_file)
 
         scfg_dict = {}
+
+        mistyped_cfg_file = os.path.join(user_home,
+                                          ".codechecker.password.json")
+
         if os.path.exists(session_cfg_file):
+            check_file_owner_rw(session_cfg_file)
             scfg_dict = load_json_or_empty(session_cfg_file, {},
                                            "user authentication")
             scfg_dict['credentials'] = \
                 simplify_credentials(scfg_dict['credentials'])
-
-            check_file_owner_rw(session_cfg_file)
-        else:
-            misstyped_cfg_file = os.path.join(user_home,
-                                              ".codechecker.password.json")
-            if os.path.exists(misstyped_cfg_file):
+            if not scfg_dict['credentials']:
+                LOG.info("No saved tokens.")
+            else:
+                LOG.debug("Tokens or passwords were found for these hosts:")
+                for k, v in scfg_dict['credentials'].items():
+                    user, _ = v.split(":")
+                    LOG.debug("  user '%s' host '%s'", user, k)
+        elif os.path.exists(mistyped_cfg_file):
                 LOG.warning("Typo in file name! Rename '%s' to '%s'.",
-                            misstyped_cfg_file, session_cfg_file)
+                            mistyped_cfg_file, session_cfg_file)
+        else:
+            LOG.info("Password file not found, checking for valid sessions.")
```

Comment:
Below you log almost the same `checking for valid sessions`. It is a little bit redundant for me.